README.md

```diff
@@ -94,25 +94,70 @@ This application relies on the following APIs:
 ## APIs
 ___
 This application creates the following API endpoints:
-🗂️ Root Directory: {deployment address tbd}
+🗂️ Root Directory: www.correct-or-crass.com / www.correctorcrass.com
 
 <br>
 
 🔠
 **Words**
+___
+- Returns a correct answer, 4 words (1 of which is the correct answer) and a photo related to the correct answer
+- Photo contains the image's url, the image owner's name and account link to enable front end to credit image owner if desired
 
-*'GET /api/v1/???'*
+*'GET /api/v1/game_play_apis'*
 
 Successful response:
 
 ```
 {
   "data": {
+      "attributes": {
```

Comment:
should "type" be a second level key along with "attribute" as we're following JSON:API principles?

Reply:
Yes, I'll add "type": "gameplay" with my next commit